deepchem/feat/graph_features.py

```diff
@@ -242,8 +241,15 @@ def bond_features(bond, use_chirality=False):
   ----------
   use_chirality: bool, optional
     If true, use chirality information.
+
```

Comment:
I think this function is important for graph model's featurizer.
it is better to add returns doc string.

Reply:
Added in a return